chips/cc26x0/src/uart.rs

```diff
@@ -264,6 +463,10 @@ impl peripheral_manager::PowerClient for UART {
     }
 
     fn lowest_sleep_mode(&self) -> u32 {
-        chip::SleepMode::DeepSleep as u32
+        if self.rx_remaining_bytes.get() == 0 {
+            chip::SleepMode::DeepSleep as u32
+        } else {
+            chip::SleepMode::Sleep as u32
+        }
```

Comment:
How does it work if the MCU goes into deep sleep, can it still receive and wake up?

Reply:
Not right now. I tried to implement a mod to `aon.rs` and I couldn't get that mod to work. For the interim I'm just going to leave it at Sleep (which can receive) until we figure out a way to get waking from deep sleep working.